sonar/contracts.py

```diff
@@ -142,36 +149,41 @@ def get_transaction(self,from_addr,value=None):
         transact_raw = self.contract.transact(txn)
         return transact_raw
 
-    def submit_model(self,model):
+    def submit_model(self, model):
         """This accepts a model from syft.nn, loads it into IPFS, and uploads
         the IPFS address to the blockchain.
 
         TODO: use best practices for storing IPFS addresses on the blockchain.
-
-        """
-
         ipfs_address = self.ipfs.add_pyobj(model.syft_obj)
-        deploy_trans = self.get_transaction(model.owner,value=self.web3.toWei(model.bounty,'ether')).addModel([ipfs_address[0:32],ipfs_address[32:]],model.initial_error,model.target_error)
-        return self.call.getNumModels()-1
+        deploy_trans = self.get_transaction(model.owner,
+        value=self.web3.toWei(model.bounty,'ether'))
+        .addModel([ipfs_address[0:32], ipfs_address[32:]], model.initial_error,
+        model.target_error)"""
```

Comment:
These vars are not even used... should they be?

Reply:
Not sure if the variable is used but the method call `.addModel` needs to happen